lib/broccoli/ember-app.js

```diff
@@ -84,7 +84,7 @@ class EmberApp {
    - trees, defaults to `{}`
    - jshintrc, defaults to `{}`
    - vendorFiles, defaults to `{}`
-   - addons, defaults to `{ blacklist: [], whitelist: [] }`
+   - addons, defaults to `{ exclude: [], include: [] }`
```

Comment:
Do we want to keep `blacklist` and `whitelist` mentioned in the docs here?

Reply:
No, I think what you have here is good